code/mathematical-algorithms/binomial_coefficient/BinomialCoefficient.py

```diff
@@ -0,0 +1,14 @@
+def binomialCoeff(n, k):
```

Comment:
Add OpenGenus comment.

Reply:
Comment is  :-  /* Part of Cosmos by OpenGenus Foundation */ 👍

Reply:
@yatharthshahjpr Kindly note your comment format is misleading.

Reply:
@yatharthshahjpr Kindly note your comment format is misleading